packages/shared/src/modules/users/libs/types/user-auth-response-dto.type.ts

```diff
@@ -5,6 +5,7 @@ type UserAuthResponseDto = {
 	firstName: string;
 	id: number;
 	lastName: string;
+	nickname: string;
```

Comment:
```suggestion
	nickname: string | null;
```
Change type here, because you have nickname as string or null in Entity